fsspec/core.py

```diff
@@ -456,6 +456,8 @@ def open(
     - For implementations in separate packages see
       https://filesystem-spec.readthedocs.io/en/latest/api.html#other-known-implementations
     """
+    kw = {"expand": False}
+    kw.update(kwargs)
```

Comment:
i would use dict.setdefault in the future. :)

Reply:
Following the conversation, `expand` should probably be promoted to the signature of open() to help those wanting to be explicit.